libs/metrics/top_level_metrics.py

```diff
@@ -98,8 +97,12 @@ def calculate_metrics_for_timeseries(
 
     icu_capacity_ratio = icu_capacity.calculate_icu_capacity(data)
 
-    vaccines_initiated_ratio = ratio_vaccinated.calculate_ratio_initiated(timeseries)
-    vaccines_completed_ratio = ratio_vaccinated.calculate_ratio_completed(timeseries)
+    vaccines_initiated_ratio = common_df.get_timeseries(
+        timeseries.date_indexed, CommonFields.VACCINATIONS_INITIATED_PCT, EMPTY_TS
+    )
+    vaccines_completed_ratio = common_df.get_timeseries(
+        timeseries.date_indexed, CommonFields.VACCINATIONS_COMPLETED_PCT, EMPTY_TS
```

Comment:
I should've asked this before, but are these fields truly percentages or are they actually ratios (22.3 or 0.223)? I would like to be specific so that we don't accidentally have confusion there.

Reply:
If they are actually ratios, I'd prefer to change the common field to say as much

Reply:
ah it looks like they are all real percentages.  Somewhere along the line we need to scale that value by 100

Reply:
Done